config/routes.rb

```diff
@@ -4,4 +4,6 @@
   resources :recipes, except: %i[edit update]
   resources :foods, except: %i[edit update]
   devise_for :users
+
+  get '/public_recipes', to: 'recipes#list_public_recipes'
```

Comment:
If we choose to change the action name, we would need to update this line to `get '/public_recipes', to: 'recipes#public_recipes'`